flatpak/manifest_tool/tests/flutter/test_helpers.py

```
@@ -0,0 +1,186 @@
+    assert result.changed
+
+    lotti = next(m for m in document.data["modules"] if m["name"] == "lotti")
+    commands = lotti.get("build-commands", [])
+
+    helper_cmd = next((cmd for cmd in commands if "setup-flutter.sh" in cmd), None)
+    assert helper_cmd is not None
+    assert "-r /custom/path1" in helper_cmd
+    assert "-r /custom/path2" in helper_cmd
+
+
+def test_bundle_app_archive_updates_sources(make_document, tmp_path: Path):
```

Ruff (0.13.1)

Unused function argument: `tmp_path`

(ARG001)